people.json

```diff
@@ -340,8 +340,10 @@
     "links": {
       "GitHub": "https://github.com/tbhaxor",
       "Twitter": "https://twitter.com/tbhaxor",
-      "LinkedIn": "https://linkedin.com/in/tbhaxor"
+      "LinkedIn": "https://linkedin.com/in/tbhaxor",
+      "Website": "https://tbhaxor.com/?source=serenity",
     }
+    "contributor": true
```

Comment:
AFAICT you have made [one commit](https://github.com/SerenityOS/serenity/commits?author=tbhaxor) about a year ago replacing a total of three words? This seems like a stretch...

Reply:
Cool I will be updating it.

Reply:
@linusg Fixed